### ISSUE TITLE
Make it possible to set "Authentification traditionnelle Wordpress" using wp_cli

### ISSUE THREAD

**Issue by domq** [epfl-sti/wordpress.plugin.tequila#2]

**Comment by domq** [comment]
It already works as-is, because there is one single option to be set in the plugin.

E.g. updating `wp_options` like this,
```
a:1:{s:14:"has_dual_auth";s:5:"false";}
```

does the right thing. It would be better not to use a structured entry in `wp_options`, but to have each setting have its own instead.

**Comment by domq** [comment]
Fixed in v0.3

```
make exec
wp --path=/srv/test/localhost/htdocs/sti/ option update plugin:epfl_tequila:has_dual_auth 1
```
